ui/src/main/resources/edu/wpi/grip/ui/templates/java/Pipeline.vm

```diff
@@ -0,0 +1,93 @@
+#**
+This creates the complete java pipeline.
+*##set($needsMut = false)
+#set($interpolation = false)
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.HashMap;
+
+import org.opencv.core.*;
+import org.opencv.core.Core.*;
+import org.opencv.features2d.FeatureDetector;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.*;
+
+/**
+* Pipeline class.
+*
+* <P>Autogenerated pipeline from GRIP.
```

Comment:
Should have a warning in here that the entire file will be regenerated and any changes to it will be lost.
Or do something like RobotBuilder and organize stuff into blocks bounded by 'Generated code' comments